src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -533,6 +533,22 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
    </CompileDependsOn>
  </PropertyGroup>
 
+<PropertyGroup >
+  <CoreCompileDependsOn>UpdateGeneratedFiles;$(CoreCompileDependsOn)</CoreCompileDependsOn>
+</PropertyGroup>
+
+<PropertyGroup >
+  <CoreCompileDependsOn>UpdateGeneratedFiles;$(CoreCompileDependsOn)</CoreCompileDependsOn>
+</PropertyGroup>
+
+<PropertyGroup >
+  <CoreCompileDependsOn>UpdateGeneratedFiles;$(CoreCompileDependsOn)</CoreCompileDependsOn>
+</PropertyGroup>
+
+<PropertyGroup >
+  <CoreCompileDependsOn>UpdateGeneratedFiles;$(CoreCompileDependsOn)</CoreCompileDependsOn>
+</PropertyGroup>
```

Comment:
no idea why this is showing up 4 times.. its only in the file once on disk and when you "view" the whole file in GitHub

Reply:
@dellis1972: How are you:

> "view[ing]" the whole file in GitHub

?

It's repeated 4 times here: https://github.com/xamarin/xamarin-android/blob/6678747758e2835383d7a7fc7b114525b6c7e46b/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L536-L550